class/defaults.yml

```
@@ -55,3 +55,16 @@ parameters:
             - alertname
     alerts:
       ignoreNames: []
+
+    silence:
+      schedule: '0 */4 * * *'
+      serviceAccountName: prometheus-k8s
+      servingCertsCABundleName: serving-certs-ca-bundle
+      jobHistoryLimit:
+        failed: 3
+        successfull: 3
```

The typo is also here, I'd suggest to rename the parameter completely
```suggestion
        successful: 3
```